planning/mission_planner/src/lanelet2_plugins/default_planner.cpp

```diff
@@ -295,17 +284,20 @@ bool DefaultPlanner::check_goal_footprint(
     lanelet::ConstLanelet combined_lanelets =
       combine_lanelets_with_shoulder(lanelets, shoulder_lanelets_);
 
-    // if next lanelet length longer than vehicle longitudinal offset
+    // if next lanelet length is longer than vehicle longitudinal offset
     if (vehicle_info_.max_longitudinal_offset_m + search_margin < next_lane_length) {
       next_lane_length -= lanelet::utils::getLaneletLength2d(next_lane);
-      boost::geometry::intersection(
-        goal_footprint, combined_lanelets.polygon2d().basicPolygon(), points_intersection);
-      if (points_intersection.empty()) {
```

Comment:
@ismetatabay 
If you have any reason to use `intersection`, please let us know.

https://github.com/autowarefoundation/autoware.universe/pull/2088